### DISCUSSION TITLE
🚀 Feature Request: Add a Counter for Taxonomy Filter widget (and all future Filter widgets)

### DISCUSSION THREAD

**Discussion by Joliamendi** [elementor/elementor#23030] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The new taxonomy widget is really cool. But it will be better if you add the counter feature for it. To display the number of posts/products in each taxonomy.

If you're adding this feature, please keep a few things in mind.

1- width and height control (For fixed width and height in different numbers. For example, the box counter number 1 should not be smaller than the number 100)
2- background-color
3- border and border-radius
4-  Prefix and Suffix )(
5- The possibility of displaying the counters in a row in front of the title (according to the image below)

**By combining items 1, 2 and 3, we can customize the counters as shown in the image below. (If needed)**

![c](https://github.com/elementor/elementor/assets/103376243/707217da-33c8-4baa-abd9-9374776e01ce)

### Describe the solution you'd like

![counter](https://github.com/elementor/elementor/assets/103376243/58ca50fb-2454-4803-ad0b-ed51c2c8be23)

**Comment by nicholaszein** [comment]
Hello @Joliamendi,

Thanks for contributing your suggestion! 🙏

Your ideas and input as a Web Creator are valuable to us. We're always keen to explore ways to enhance Elementor to meet our community's needs better.

### 💡 Due to the high volume of feature requests we receive and our commitment to continuous improvement, we can't guarantee that every suggestion will be immediately included in our roadmap.

We appreciate your patience as we take the time to review your request thoroughly. We aim to ensure all decisions align with our mission to provide Web Creators with the tools they need to build their dream websites.

Kind regards

**Comment by toltechinternet** [comment]
Does anyone know how to do this using custom code? I am currently building a shop that uses the loop builder for displaying products and the taxonomy filter for displaying the categories. I would love to simply have an item count for each category to show the end-user that there is X amount of products in each category.

Any help would be greatly appreciated.

**Reply by DanielCyran** [reply]
@nicholaszein     Over the past few months, Elementor has been doing an excellent job in terms of development, particularly focusing on the speed of website performance. However, we must also acknowledge the concern raised that such features are extremely important for online stores. Currently, we are facing issues with filtering in stores and unattractive product image layouts, as well as the outdated product rating widget. It would be great if Elementor continued to focus on improving both speed and online store functionalities. We greatly appreciate your work (we have 1,000 Elementor licenses and urgently need the development of store widgets).
